### PR TITLE
pr for microverse tse approval: milestone 3 interactive form

### PR DESCRIPTION
**Pull request by firmoholanda**
in this pr, we implemented the folowing:

- make the BooksForm component stateful;
-  implement the handleChange;
- mplement handleSubmit;
- connect the REMOVE_BOOK action to the component;
- add a new table column to the BooksList component;
- implement the handleRemoveBook;
- add uuid to book id.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -1,20 +1,26 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Book = ({ book }) => (
+const Book = ({ book, handleRemoveBook }) => (
   <tr>
-    <td>{book.id}</td>
+    <td>{book.id.slice(0, 5)}</td>
```
Comment (bellom) [inline src/components/Book.js:6]: I'm curious, why are you limiting the id to `1-5`?

File: src/components/Book.js [hunk 2]
```diff
@@ -1,20 +1,26 @@
...
+      <button onClick={() => handleRemoveBook(book)} className="btn btn-danger btn-sm" type="button">remove book</button>
+      {' '}
+    </td>
   </tr>
 );
 
 Book.propTypes = {
   book: PropTypes.shape({
-    id: PropTypes.number.isRequired,
+    id: PropTypes.string.isRequired,
```
Comment (bellom) [inline src/components/Book.js:19]: you change the `id` to a string - could this be because of the `uuid`?

**Review by bellom — APPROVED** [review]
## Hi Team

##### You have done wonderfully well, you passed all checklist for this milestone :+1:
 
Let's move on to the next milestone.

Good luck with your next milestone :slightly_smiling_face:_ 

**Best,**

**Oluwaseun J. Bello**
**_Slack Handle - @sean bellom_**

File: src/components/Book.js (hunk 1 above)
Comment (firmoholanda) [reply src/components/Book.js:6]: :) hi there bellom. we sliced to 5 char in order to "look better" on the html table for ms 3. but, we will remove this and let the full length of uuid on next mss.

**Pull request merged**